packages/template/vite-typescript/test/ViteTypeScriptTemplate_spec_slow.ts

```diff
@@ -21,7 +21,8 @@ describe('ViteTypeScriptTemplate', () => {
   after(async () => {
     await yarnOrNpmSpawn(['link:remove']);
     await killWindowsEsbuildExe();
-    await fs.remove(dir);
+    // TODO: use the async API
```

Comment:
If possible, please describe why the async API cannot be used here temporarily, and attach your github id after the TODO comment:

```js
// TODO: foo bar @YOUR_GITHUB_ID
```

Reply:
Resolved 👉 https://github.com/electron/forge/pull/3336/commits/32e683ea4cc6490336f44072c1ce5efef3bfb20c